changes/ce/fix-12996.en.md

```diff
@@ -0,0 +1 @@
+Fix process leak in `emqx_retainer` application. Previously, client disconnection while receiving retained messages could cause a process leak.
```

Comment:
does the process leak indefinitely ?

Reply:
Dispatching is done via permanent process pool, so pool processes hold leaked qlc handlers forever.